### PR TITLE
Disable API check for newly added packages

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Mvc.RazorPages/Microsoft.AspNetCore.Mvc.RazorPages.csproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;aspnetcoremvc;cshtml;razor</PackageTags>
+    <EnableApiCheck>false</EnableApiCheck>
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Mvc.RazorPages/Microsoft.AspNetCore.Mvc.RazorPages.csproj]: We should have an empty baseline rather then disabling API check. Then when we go to update all of the baselines the empty one will become full

Comment (Eilon) [inline src/Microsoft.AspNetCore.Mvc.RazorPages/Microsoft.AspNetCore.Mvc.RazorPages.csproj]: Oh right, that's what we decided to do. I totally forgot.

Comment (rynowak) [inline src/Microsoft.AspNetCore.Mvc.RazorPages/Microsoft.AspNetCore.Mvc.RazorPages.csproj]: ...

Comment (pranavkm) [inline src/Microsoft.AspNetCore.Mvc.RazorPages/Microsoft.AspNetCore.Mvc.RazorPages.csproj]: Bad @Eilon

**Comment by pranavkm** [comment]
🆙 📅

**Pull request merged**